### ISSUE TITLE
How do I increase the MAX CONTENT size? I increased max file size and memory usage in the compose, but the file still failed...

### ISSUE THREAD

**Issue by cleverestx** [aschmelyun/subvert#37]
I almost ran a 2.5GB MP4 file though but I was some amount of just BYTES off from being able to complete it. How do I fix this? Both other sizes were set to 4096M

_*Edit it IS reading and outputting MP3 (songs) fine...lyrics and summary, but no video is working, see below*_

Thank you.

**Comment by cleverestx** [comment]
I increased both settings to 6144M as per above, but it doesn't seem to matter. Content size is failing this file. 
![image](https://github.com/aschmelyun/subvert/assets/28905253/b1bd94f3-c034-479d-847d-5cf8f26a05d1)

**Comment by cleverestx** [comment]
Hmmm...I guess something is broken? I tried a 1.18GB File as a text (smaller file) and I get the same error with almost the same content size limit listed.  How do I resolve this? Thanks.

![image](https://github.com/aschmelyun/subvert/assets/28905253/039b8137-598c-42e5-932e-19b01330da3e)

**Comment by cleverestx** [comment]
Confirmed, broken (for whatever reason), as it happens even on a 385M Mp4 file. (and even a 141MB one!) - Any help is appreciated. I really want this to work...

![image](https://github.com/aschmelyun/subvert/assets/28905253/7f22e370-53c3-47e3-a19e-901c0d6060da)

**Comment by cleverestx** [comment]
Still hopeful this can be improved...please let me know if I can help somehow.

**Comment by cleverestx** [comment]
Looking forward to an update on this!

**Comment by Swedlund** [comment]
I'm having the same issue.
@cleverestx, did you find a way to fix this?

**Comment by cleverestx** [comment]
No, I was told by the author that this is something that he would look into, and he seemed interested in fixing it...but I haven't gotten any later updates on it. I really do hope it happens.

**Comment by Swedlund** [comment]
> No, I was told by the author that this is something that he would look into, and he seemed interested in fixing it...but I haven't gotten any later updates on it. I really do hope it happens.

Ok, thx for letting me know.

**Comment by aschmelyun** [comment]
@Swedlund @cleverestx Updates are coming on this within the next month or so, sorry for the delay!

**Comment by cleverestx** [comment]
> [@Swedlund](https://github.com/Swedlund) [@cleverestx](https://github.com/cleverestx) Updates are coming on this within the next month or so, sorry for the delay!

Could this possibly happen..soon? Would be so awesome....
